### ISSUE TITLE
Use expansion with *args

### ISSUE THREAD

**Issue by Sergeileduc** [Azrood/DCTbot#38]
In line :

https://github.com/Azrood/DCTbot/blob/dc7ff70ba45befaa29ae2c3a58b44e6069eccbaf/bot.py#L543

your should use expansion (*args) :
like : https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#variable

`asyn def foo(ctx, *args):`

In doing so, args will be a list (of variable lenght).
That is what you want to do.

The way you did it (with keyword-only argument), ie `def foo(ctx, date, *, args="")`
puts all your parameters (after date) in a single string.

And then you need to split it :
https://github.com/Azrood/DCTbot/blob/dc7ff70ba45befaa29ae2c3a58b44e6069eccbaf/utils/tools.py#L54

So basically, what you do is forcing python to make a single string, and not a list.
And then, split that string, to get back to a list...

:rofl:

**Comment by Azrood** [comment]
Gotta admit, it wasn't my brightest idea :)
